projects/js-packages/publicize-components/src/components/manage-connections-modal/confirmation-form/index.tsx

```diff
@@ -199,6 +202,23 @@ export function ConfirmationForm( { keyringResult, onComplete, isAdmin }: Confir
 							'jetpack'
 						) }
 					</p>
+					{ keyringResult?.show_linkedin_warning && (
+						<Notice type={ 'warning' }>
+							<p>
+								{ __(
+									'We cannot retrieve which company pages you have access to. This is a known issue with the LinkedIn API. If you would like to connect a company page, please retry after 5 minutes. For more information, click',
```

Comment:
Probably "could not" instead of "cannot"?

Also, we should not break/split i18n strings, i.e. here `click` and `here` are in separate strings which is noy easy to join with context in many languages.

You can either use `<a>` and wrap the full string `click here` or simply replace the last sentence with a "Learn more" hyperlink.